frontend/src/__tests__/api.test.js

```diff
@@ -54,6 +54,94 @@ expect.extend({
     }
 });
 
+/**
+ * Seeding the database with the minimal set of API calls to create an assignment.
+ */
+function dataBaseSeed(api = { apiGET, apiPOST }) {
+    const { apiGET, apiPOST } = api;
+    let session = null;
+
+    beforeAll(async () => {
+        await apiPOST("/admin/debug/snapshot");
+        await apiPOST("/admin/debug/clear_data");
+    }, 15000);
+
+    //Create Session
+    const minimalSessionData = {
+        start_date: new Date("2020/02/10").toISOString(),
+        end_date: new Date("2020/12/31").toISOString(),
+        name: "Minimal Session Data",
+        rate1: 50
+    };
+
+    it("create a session", async () => {
+        const resp1 = await apiPOST("/admin/sessions", minimalSessionData);
+        const { payload: minimalSession } = await apiGET("/admin/sessions");
```

Comment:
You need `minimalSession` for the ID, but you're storing this as a local variable...Also, `resp1` should contain the session that was just created, so there is no need for a separate GET.